schema.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(225),
+    date_of_birth DATE
+);
+CREATE INDEX idx_patients_id ON patients(id); 
+
+
+CREATE TABLE medical_histories(
+    id INT  PRIMARY KEY,
+    admited_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(225)
+);
+CREATE INDEX idx_medical_histories_id ON medical_histories(id); 
+
+<<<<<<< HEAD
+
+=======
+>>>>>>> e2bae7a56d917c1421442574c618388b94874ccd
+CREATE TABLE invoices (
+    id INT PRIMARY KEY,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INT REFERENCES medical_histories(id)
+);
+CREATE INDEX idx_invoices_id ON invoices(id); 
+
+CREATE TABLE treatments(
+    id INT PRIMARY KEY,
+    type VARCHAR(225),
+    name VARCHAR(225)
+);
```

Comment:
- [ ] As mentioned by the previous :point_up: reviewer if you look at the schema provided for this milestone, you will notice that the `medical_histories` table and the `treatment` table have a many-to-many relationship. I would advise you to create a join table `medical_histories_has_treatments` with the `ID` column and foreign keys needed in a join table but still you haven't implemented this change kindly create a join table `medical_histories_has_treatments` with the `ID` column and foreign keys :slightly_smiling_face: